src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -250,5 +269,129 @@ public static string GetInsertionText(CompletionItem item)
             item.Properties.TryGetValue("InsertionText", out var text);
             return text;
         }
+
+        public static CompletionItem CreateWithSymbolId(
+            string displayText,
+            IReadOnlyList<ISymbol> symbols,
+            int contextPosition,
+            string sortText = null,
+            string insertionText = null,
+            Glyph? glyph = null,
+            string filterText = null,
+            int? matchPriority = null,
+            SupportedPlatformData supportedPlatforms = null,
+            ImmutableDictionary<string, string> properties = null,
+            ImmutableArray<string> tags = default(ImmutableArray<string>),
+            CompletionItemRules rules = null)
+        {
+            return CreateWorker(displayText, symbols, AddSymbolEncoding, contextPosition, sortText, insertionText, glyph,
+                          filterText, matchPriority, supportedPlatforms, properties, tags, rules);
+        }
+
+        public static CompletionItem CreateWithSymbolId(
```

Comment:
There are lots of callers of both overloads: some providers wind up with a list of symbols collapsed into one item and some don't.